workers/ohsome_quality_analyst/cli.py

```diff
@@ -147,12 +149,8 @@ def create_indicator(
             feature["properties"].update(vars(indicator.result))
         if outfile is None:
             outfile = infile.stem + "_" + indicator_name + infile.suffix
```

Comment:
I'm not sure but this looks a bit hacky to me. Maybe add a comment here in the code how the outfile name will look like based on what is provided as input file.

Reply:
this is included for a while now, not with this PR. It was the best way @matthiasschaub and I found to automatically create an outputfile definition which works both on linux and windows, due to the different handling of subdirs. But I'll add a comment what is done here... alternatively, I could also change it that an output geojson will only be written if the outfile is defined by the user

Reply:
I think it's fine if you add a comment stating just what you explained to me here. That you do it like this with good reasons.

Reply:
is the comment I added understandable?

Reply:
yes